File: show-case/jpa-show-case/src/main/java/com/yicj/study/jpa/service/UserInfoService.java
package com.yicj.study.jpa.service;


import com.yicj.study.jpa.entity.UserInfo;

import java.util.List;

/**
 * <p>
 * UserInfoService
 * </p>
 *
 * @author yicj
 * @since 2024/09/04 13:59
 */
public interface UserInfoService {

    List<UserInfo> listAll() ;

    void deleteById(String id) ;

    String save(UserInfo userInfo) ;
}

File: show-case/mybatis-show-case/src/main/java/com/yicj/study/mybatis/service/UserInfoService.kt
package com.yicj.study.mybatis.service

import com.baomidou.mybatisplus.extension.service.IService
import com.yicj.study.mybatis.entity.AccountInfo
import com.yicj.study.mybatis.entity.UserInfo
import reactor.core.publisher.Mono

interface UserInfoService : IService<UserInfo>{
    fun saveUserAndAccount(user: UserInfo, account: AccountInfo) : String?

    fun saveUserAndAccount2(user: UserInfo, account: AccountInfo) : String?

    fun saveUserAndAccountMono(user: UserInfo, account: AccountInfo) : String?

}